app/queries/news_items/with_filters.rb

```diff
@@ -1,4 +1,5 @@
 module NewsItems
+  NUMBER_OF_TOP_COINS = 
```

Comment:
Er what's going on here? Doesn't look like there's a right hand side assignment to the `NUMBER_OF_TOP_COINS` constant, unless I'm missing something?

Reply:
Nope was going to take it out then decided it was unnecessary. Didn't realized that got left there :disappointed: